tools/yaml-templates/ios-test.yml

```diff
@@ -80,7 +80,10 @@ steps:
       -sdk iphonesimulator
       -parallel-testing-enabled YES
       -parallel-testing-worker-count 2
-      -destination 'platform=iOS Simulator,name=iPhone 14,OS=16.2'
+      -destination 'platform=iOS Simulator,name=iPhone 14 Pro,OS=16.2'
+      -screenshot-enabled="YES"
```

Comment:
Are screenshots only taken on failures or are they taken for every test?

Reply:
By default it is only taken when the case fails.